resources/config/wdio.prod.browserstack.conf.ts

```diff
@@ -3,78 +3,99 @@ import * as _ from "lodash";
       os_version: "Catalina",
       browserName: "Chrome",
       browser_version: "latest",
-      acceptInsecureCerts: true,
-      name:
-        parseArgs(process.argv.slice(2))["bstack-session-name"] ||
-        "default_name",
-      build:
-        process.env.BROWSERSTACK_BUILD_NAME ||
-        "webdriver-io-boilerplate" + " - " + epochTime,
     },
     {
       os: "OS X",
       os_version: "Catalina",
       browserName: "Firefox",
       browser_version: "latest",
-      acceptInsecureCerts: true,
-      name:
-        parseArgs(process.argv.slice(2))["bstack-session-name"] ||
-        "default_name",
-      build:
-        process.env.BROWSERSTACK_BUILD_NAME ||
-        "webdriver-io-boilerplate" + " - " + epochTime,
+    },
+    {
+      os: "OS X",
+      os_version: "Catalina",
+      browserName: "Firefox",
+      browser_version: "latest",
+    },
+    {
+      os_version: "10.0",
+      device: "Samsung Galaxy S20",
+      real_mobile: "true",
+      browserName: "Android",
+    },
+    {
+      os_version: "11.0",
+      device: "Google Pixel 4",
+      real_mobile: "true",
+      browserName: "Android",
+    },
+    {
+      os_version: "11.0",
+      device: "Google Pixel 5",
+      real_mobile: "true",
+      browserName: "Android",
     },
   ],
-  afterTest: function (
+  afterTest: async function (
     test: { title: string },
-    context: Record<string, unknown>,
+    _context: Record<string, unknown>,
     { passed }: Record<string, unknown>
   ) {
     if (parseArgs(process.argv.slice(2))["bstack-session-name"]) {
-      browser.executeScript(
+      await browser.executeScript(
         'browserstack_executor: {"action": "setSessionName", "arguments": {"name":"' +
           parseArgs(process.argv.slice(2))["bstack-session-name"] +
           '" }}'
       );
     } else {
-      browser.executeScript(
+      await browser.executeScript(
         'browserstack_executor: {"action": "setSessionName", "arguments": {"name":"' +
           test.title +
           '" }}'
       );
     }
 
     if (passed) {
-      browser.executeScript(
+      await browser.executeScript(
         'browserstack_executor: {"action": "setSessionStatus", "arguments": {"status":"passed","reason": "Assertions passed"}}'
       );
     } else {
-      browser.takeScreenshot();
-      browser.executeScript(
-        'browserstack_executor: {"action": "setSessionStatus", "arguments": {"status":"failed","reason": "At least 1 assertion failed."}}'
+      await browser.takeScreenshot();
+      await browser.executeScript(
+        'browserstack_executor: {"action": "setSessionStatus", "arguments": {"status":"failed","reason": "At least 1 assertion failed." }}'
       );
     }
   },
 };
 
 export const config = _.defaultsDeep(overrides, defaultConfig);
+
+config.capabilities.forEach(function (caps: { [x: string]: unknown }) {
+  for (const i in config.commonCapabilities)
+    caps[i] = caps[i] || config.commonCapabilities[i];
+});
```

Comment:
can we refactor this? It looks weird to export something and then change its value